### PR TITLE
Minimal working example for usage in slurm

### PR DESCRIPTION
**Pull request by exaexa**

### CODE REVIEW

**Comment by exaexa** [comment]
Closes #18

File: docs/src/slurm.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+   performance), load more data in the memory (usually, much more total memory
+   is available in the clusters than on a single computer), but keep an eye on
+   the communication overhead, transferring only the minimal required amount of
+   data as seldom as possible.
+
+### Preparing the packages
+
+The easiest way to install the packages is using a single-machine interactive job. On the access node of your HPC, run:
+```sh
+srun --pty -n1 -c1 -t60 --mem 1G /bin/bash
```
Comment (laurentheirendt) [inline docs/src/slurm.md:36]: Maybe you can use the interactive partition to get a faster allocation:
```
srun -p interactive --pty bash -i
```

File: docs/src/slurm.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+```
+
+When the shell opens (the prompt should change), you can load the Julia module,
+usually with a command such as this:
+
+```sh
+module load lang/Julia/1.3.0
```
Comment (laurentheirendt) [inline docs/src/slurm.md:43]: Maybe add that the user can find which Julia module to load by:
```
module spider julia
```

File: docs/src/slurm.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+- the job will be visible in the queue as `MyDistributedJob`
+- it will load Julia 1.3.0 module on the workers, so that `julia` executable is
+  available (you may want to consult the versions availability with your HPC
+  administrators)
+- finally, it will run the Julia script `run-analysis.jl`
+
+### Julia script
+
+The `run-analysis.jl` may look as follows:
+```julia
```
Comment (laurentheirendt) [inline docs/src/slurm.md:99]: Is this script somewhere I can download?

File: docs/src/slurm.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+# do not forget to save the results!
+f = open("result.txt", "w")
+println(f, totalResult)
+close(f)
+```
+
+Finally, you can execute the whole thing with `sbatch`:
+```sh
+sbatch run-analysis.batch
```
Comment (laurentheirendt) [inline docs/src/slurm.md:124]: Maybe add if this command should be launched from the access or the compute node

**Review by laurentheirendt — CHANGES_REQUESTED** [review]

File: docs/src/slurm.md (hunk 1 above)
Comment (exaexa) [reply docs/src/slurm.md:36]: not sure if `-p interactive` is portable, but I simplified and explained the command.

File: docs/src/slurm.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+- the job will be visible in the queue as `MyDistributedJob`
+- it will load Julia 1.3.0 module on the workers, so that `julia` executable is
+  available (you may want to consult the versions availability with your HPC
+  administrators)
+- finally, it will run the Julia script `run-analysis.jl`
+
+### Julia script
+
+The `run-analysis.jl`
+([download](https://github.com/LCSB-BioCore/DistributedData.jl/blob/master/docs/slurm-example/run-analysis.jn))
```
Comment (laurentheirendt) [inline docs/src/slurm.md:108]: `.jl` no?

Comment (exaexa) [reply docs/src/slurm.md]: yes. :rofl: Fixed.

**Review by laurentheirendt — APPROVED** [review]

**Pull request merged**